readme.md

```diff
@@ -18,13 +18,13 @@ $ npm install --save-dev gulp-babel babel-preset-es2015
 const gulp = require('gulp');
 const babel = require('gulp-babel');
 
-gulp.task('default', () => {
-	return gulp.src('src/app.js')
```

Comment:
I've removed curly braces also. This is advantage of the arrow syntax: `() => 42` is the same as `() => { return 42; }`